client/hotrod-client/src/main/java/org/infinispan/client/hotrod/configuration/StatisticsConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package org.infinispan.client.hotrod.configuration;
+
+import org.infinispan.commons.configuration.attributes.Attribute;
+import org.infinispan.commons.configuration.attributes.AttributeDefinition;
+import org.infinispan.commons.configuration.attributes.AttributeSet;
+import org.infinispan.commons.configuration.attributes.IdentityAttributeCopier;
+import org.infinispan.commons.jmx.MBeanServerLookup;
+import org.infinispan.commons.jmx.PlatformMBeanServerLookup;
+import org.infinispan.commons.util.Util;
+
+/**
+ * @author Tristan Tarrant
+ * @since 9.4
+ */
+public class StatisticsConfiguration {
+   public static final AttributeDefinition<Boolean> ENABLED = AttributeDefinition.builder("enabled", false).immutable().build();
+   public static final AttributeDefinition<Boolean> JMX_ENABLED = AttributeDefinition.builder("jmx_enabled", false).immutable().build();
```

Comment:
Yes, embedded is broken :) I want to eventually use the attribute names in an automatic Properties serializer / parser so that it doesn't need a hand-written one.

Reply:
What do underscores change about that? Properties are case sensistive, too. And if you have any religious problem with uppercase, I think that dashes are nicer than underscores (and the XML word seems to share that feeling).